nnef/src/resource.rs

```diff
@@ -0,0 +1,119 @@
+use std::path::Path;
+
+use crate::ast::{Document, QuantFormat};
+use crate::internal::*;
+use tract_core::downcast_rs::{impl_downcast, DowncastSync};
+
+pub const NNEF_DOCUMENT_FILE: &'static str = "graph.nnef";
```

Comment:
If we're isolating these in constants, then I would prefer them to be GRAPH_NNEF_FILENAME and GRAPH_QUANT_FILENAME.

Reply:
👍